docs/6-rag/5-rag-pipelines.md

```diff
@@ -0,0 +1,119 @@
+# 🌳 Automating RAG with KFP Pipelines
+
+Your document intelligence RAG system works brilliantly in notebooks, but what happens when you need to process hundreds of research papers for your educational platform? 
+Manual execution doesn't scale, and RDU needs reliable, always-available intelligent document processing.
+
+## 🔍 What is Kubeflow Pipelines (KFP)?
```

Comment:
We will already have covered KFP before this so can skip this section to slim it down a bit if you want